ext/mbstring/mbstring.c

```diff
@@ -3471,19 +3511,16 @@ static bool mb_recursive_convert_variable(zval *var, const mbfl_encoding* from_e
 	return false;
 }
 
-/* {{{ Converts the string resource in variables to desired encoding */
 PHP_FUNCTION(mb_convert_variables)
 {
 	zval *args;
 	zend_string *to_enc_str;
 	zend_string *from_enc_str;
 	HashTable *from_enc_ht;
 	const mbfl_encoding *from_encoding, *to_encoding;
-	mbfl_encoding_detector *identd;
-	int n, argc;
+	int argc;
```

Comment:
``Z_PARAM_VARIADIC()`` should take a ``uint32_t``
```suggestion
	uint32_t argc;
```